src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,97 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
 import './FindMovie.scss';
 
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { getMovies } from '../../api/api';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = ({ addMovie }) => {
+  const [inputValue, setValue] = useState('');
+  const [movie, setMovie] = useState('');
+
+  const fetchMovie = async() => {
+    const res = await getMovies(inputValue);
+
+    setMovie(res);
+  };
+
+  const addFindedMovie = (e) => {
```

Comment:
Why do you need 'event' here?

Reply:
Because I use e.preventDefault() to correct work form event onSubmit. Without it page just update without adding movie to the list

Reply:
👍